packages/mdc-snackbar/README.md

```diff
@@ -108,69 +141,164 @@ This will move the snackbar offscreen until the CSS is fully loaded and avoids a
 CSS Class | Description
 --- | ---
 `mdc-snackbar` | Mandatory. Container for the snackbar elements.
-`mdc-snackbar__action-wrapper` | Mandatory. Wraps the action button.
-`mdc-snackbar__action-button` | Mandatory. The action button.
-`mdc-snackbar__text` | Mandtory. The next of the snackbar.
-`mdc-snackbar--align-start` | Optional. Class to align snackbar to start, ltr dependent.
-`mdc-snackbar--action-on-bottom` | Optional on the mdc-snackbar element. Moves action to bottom of snackbar. Can be applied in js.
-`mdc-snackbar--multiline` | Optional on the mdc-snackbar element. Makes the snackbar multiple lines. Can be applied in js.
+`mdc-snackbar__label` | Mandatory. Message text.
+`mdc-snackbar__actions` | Optional. Wraps the action button/icon elements, if present.
+`mdc-snackbar__action-button` | Optional. The action button.
+`mdc-snackbar__action-icon` | Optional. The dismiss ("X") icon.
+`mdc-snackbar--opening` | Optional. Indicates that the snackbar is open and visible.
+`mdc-snackbar--open` | Optional. Applied automatically when the snackbar is in the process of animating open.
+`mdc-snackbar--closing` | Optional. Applied automatically when the snackbar is in the process of animating closed.
+`mdc-snackbar--leading` | Optional. Positions the snackbar on the leading edge of the screen (left in LTR, right in RTL) instead of centered.
+`mdc-snackbar--stacked` | Optional. Positions the action button/icon below the label instead of along side it. Should only be applied to devices with large screens.
+`mdc-snackbar--wide` | Optional. Increases the margins between the snackbar and the viewport. Should only be applied to devices with large screens.
+
+### Sass Mixins
+
+Mixin | Description
+--- | ---
+`mdc-snackbar-fill-color($color)` | Sets the fill color of the snackbar.
+`mdc-snackbar-label-ink-color($color)` | Sets the color of the snackbar's label text.
+`mdc-snackbar-shape-radius($radius, $rtl-reflexive)` | Sets the rounded shape to snackbar surface with given radius size. Set `$rtl-reflexive` to true to flip radius values in RTL context, defaults to false.
+`mdc-snackbar-min-width($min-width)` | Sets the minimum width of the snackbar (defaults to 344px).
```

Comment:
nit: back-ticks for default values `344px`. same below.

Reply:
Done